kubernetes.go

```diff
@@ -354,6 +365,21 @@ func (svc *KubernetesServiceOp) Get(ctx context.Context, clusterID string) (*Kub
 	return root.Cluster, resp, nil
 }
 
+// GetUser retrieves the details of a Kubernetes cluster user.
+func (svc *KubernetesServiceOp) GetUser(ctx context.Context, clusterID string) (*KubernetesClusterUser, *Response, error) {
+	path := fmt.Sprintf("%s/%s/user", kubernetesClustersPath, clusterID)
+	req, err := svc.client.NewRequest(ctx, http.MethodGet, path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	root := new(kubernetesClusterUserRoot)
+	resp, err := svc.client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
```

Comment:
Do we need to defer-close the response body, or does godo somehow does this under the hood?

Reply:
Good catch. We currently don't close the response body in any of the other handlers, leaving it up to the user to do it (I assume), so I'd be inclined to continue with that pattern.

Reply:
SGTM.